client/components/credit-card/stored-card.jsx

```diff
@@ -0,0 +1,71 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import React from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import { localize } from 'i18n-calypso';
+
+export const getCreditCardSummary = ( translate, type, digits ) => {
+	const supportedTypes = {
+		amex: translate( 'American Express' ),
+		discover: translate( 'Discover' ),
+		mastercard: translate( 'Mastercard' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **52** times:
`translate( 'MasterCard' )` *ES Score: 32*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/credit-card-list-item-into-reusable-component)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).